src/clj/rems/standalone.clj

```diff
@@ -223,6 +225,17 @@
           (when-not (validate/validate)
             (System/exit 2)))
 
+        "rename-user"
+        (let [[_ old-userid new-userid] args]
+          (if (not (and old-userid new-userid))
+            (do (usage)
+                (System/exit 1))
+            (do (println "\n\n*** Renaming a user's identity can't easily be undone. ***\nType 'YES' to proceed or anything else to run a simulation only.")
+                (let [simulate? (not= "YES" (read-line))]
+                  (println (if simulate? "Simulating only..." "Renaming..."))
+                  (mount/start #'rems.config/env #'rems.db.core/*db*)
+                  (rems.db.fix-userid/fix-all old-userid new-userid simulate?)))))
```

Comment:
we could print a reminder line to the user to reboot the server so that caches are updated (as pointed out in the changelog)

Reply:
Excellent idea!